BAZProjectC1/Components/BasicInfoMovie/backButton.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  backButton.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 28/09/22.
+//
+import UIKit
+
+class backButton: UIButton {
+    
+    public lazy var iconButton: UIImageView = {
```

Comment:
¿Es necesario que esta propiedad sea pública?, veo que no se está utilizando en algún otro archivo. 
¿Podríamos hacerla privada y de ser necesario configurarla con un modelo utilizando una función?

```suggestion
    private lazy var iconButton: UIImageView = {
```

```
    
```